R/onLoad.R

```diff
@@ -66,6 +66,16 @@
       lockBinding("rbind.data.frame",baseenv())
     }
   }
+  if (session_r_version < "3.6.0") {
```

Comment:
This should help keep the two intimately-linked pieces of logic in sync despite residing in totally different parts of the codebase

```suggestion
  if (session_r_version < "3.6.0") { # corresponds to S3method() directive in NAMESPACE
```